File: APIManga/Model/Page.cs
using APIManga.Model;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace APIManga.Model
{
	public class Page
	{
		[Key]
		public int Id { get; set; }
		public int ChapterId { get; set; }
		public Chapter Chapter { get; set; }
		public int? PageNumber { get; set; }
		public virtual List<Image> Images { get; set; }
	}
}

File: backend/app/models/page.py
from sqlalchemy import Column, Integer, ForeignKey
from sqlalchemy.orm import relationship

from app.database.connection import Base


class Page(Base):
    __tablename__ = "page"

    id = Column(Integer, primary_key=True)
    chapter_id = Column(Integer, ForeignKey("chapter.id"))
    page_number = Column(Integer, nullable=True)

    # RELAÇÕES

    # N:1 -> Page -> Chapter
    chapter = relationship(
        "Chapter",
        back_populates="pages"
    )

    # 1:N -> Page -> Images
    images = relationship(
        "Image",
        back_populates="page",
        cascade="all, delete-orphan"
    )